src/nft/hooks/useSearchHistory.ts

```diff
@@ -18,6 +19,16 @@ export const useSearchHistory = create<SearchHistoryProps>()(
           return { history: historyCopy }
         })
       },
+      updateItem: (update: FungibleToken | GenieCollection) => {
```

Comment:
is it worth ensuring that this item has more recent data before overwriting? would we ever have the opposite case happen?

Reply:
I don't think that is possible, because CollectionRow (which calls addItem) is populated by the same query that would be calling updateItem. It may be possible in the future, if this codepath grows and/or is reused.

However, even if it were possible, I think it'd be preferable to always update searchHistory when requested, so that if multiple queries are being used, searchHistory is forced to be in sync, and there is no disparity possible.